### DISCUSSION TITLE
What is the use of this tab/feature?

### DISCUSSION THREAD

**Discussion by harveyinstinct** [qnblackcat/uYouPlus#1150] [Q&A]
![IMG_1264](https://github.com/qnblackcat/uYouPlus/assets/136118388/818898d7-4626-4ed4-8ed7-1aa81141c2d3)
Could anyone explain what is the use of this feature? Can i download youtube videos directly to my phone library but if so then how? Could anyone explain? 
Thank you uYou team! Great work!

**Comment by BlueJeans7** [comment]
I wish that instead of tab, this option was present as a setting along with other settings.

That said, you can tap on the top left setting gear icon for more options.

**Comment by FabiFNA** [comment]
Yes you can download videos to that tab. On a video press the icon on the top right to download and choose the resolution/format.
